### PR TITLE
AS4

### PR DESCRIPTION
**Pull request by Isabelmgonz**
AS4

### CODE REVIEW

File: Assignment4/Isabelmgonz-190399/task06.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/github/FacultadInformatica-LinkedData/Curso2022-2023/blob/master/Assignment4/course_materials/notebooks/Task06.ipynb
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/Isabelmgonz-190399/task06.py]: won't work in Python. This is notebook magic

File: Assignment4/Isabelmgonz-190399/task07.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+"""
+
+# TO DO
+q2 = prepareQuery ('''
+    SELECT ?subj ?subj2 WHERE {
+      {
+        ?subj rdfs:type ns:Person.}
+        UNION
+        {?subc rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/Isabelmgonz-190399/task07.py]: use * to chain properties and get all sublcasses of person

File: Assignment4/Isabelmgonz-190399/task07.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+"""
+
+# TO DO
+q3 = prepareQuery ('''
+    SELECT ?subj ?property ?class WHERE { 
+      {?subj rdf:type ns:Person.
+       ?subj ?property ?class}
+       UNION 
+      {?aux rdfs:subClassOf ns:Person.
+       ?subj rdf:type ?aux.
```
Comment (dgarijo) [inline Assignment4/Isabelmgonz-190399/task07.py:85]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]
small changes are needed

**Comment by Isabelmgonz** [comment]
Fix_resolved

File: Assignment4/Isabelmgonz-190399/task07.py (hunk 3 above)
Comment (Isabelmgonz) [reply Assignment4/Isabelmgonz-190399/task07.py:85]: Resolved

**Review by dgarijo — APPROVED** [review]

**Pull request merged**